data-model/SDP-data-model-templates/manuscript-object_sdp-template.json

```diff
@@ -40,7 +37,7 @@
   },
   "events": [
     {
-      "id": "ARK",
+      "id": "URI",
```

Comment:
Events would at most have a uniquely-generated id. and maybe not even that.

In any case, events might be for a later phase entirely. I could imagine that for now we just have paratexts we record as part of the codicological or textual objects. These might have evidence for events, but maybe for now they remain wrapped up in the paratexts. Down the line we can mine these paratexts for event information.